app/main.py

```diff
@@ -1,8 +1,51 @@
-class Car:
-    # write your code here
-    pass
+class Car():
+
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        if distance_from_city_center > 0:
+            self.distance_from_city_center = distance_from_city_center
```

Comment:
The check for `distance_from_city_center` being greater than zero is good to prevent division by zero. However, if `distance_from_city_center` is zero, it should be set to a default value or raise an exception to handle this case properly.